src/test/java/org/jabref/logic/msbib/MsBibMappingTest.java

```diff
@@ -0,0 +1,19 @@
+package org.jabref.logic.msbib;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class MsBibMappingTest {
```

Comment:
Please add a test for a number not appearing in the map when executing `MSBibMapping.getLanguage`. You will never know, which language Microsoft will add in the future. The converter should also be able to handle unknown language numbers (with a best-effort?)